app/views/world_wide_taxons/accordion.html.erb

```diff
@@ -38,7 +48,24 @@
             }
           %>
         <% end %>
-        <%= render 'govuk_publishing_components/components/accordion', items: items %>
+        <div data-module="gem-track-click gtm-click-tracking">
```

Comment:
Why is `gem-track-click` being added here? That module is for the existing UA tracking, it's not related to the new GTM stuff.